Woof/Woof.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
According to the documentation, it is better to commit this file in Git to ensure that everyone in the team uses the same package dependency version. If this file is ignored,  the situation when team members can have different versions of packages as a dependency can occur, and it will be hard to find and investigate the cause of problems and reproduce bugs, for example.